### PR TITLE
add LoopSplat in SLAM

### PR DESCRIPTION
**Pull request by Zhu-Liyuan**
https://loopsplat.github.io/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3035,6 +3035,15 @@ We present Real-time Gaussian SLAM (RTG-SLAM), a real-time 3D reconstruction sys
...
+Simultaneous Localization and Mapping (SLAM) based on 3D Gaussian Splats (3DGS) has recently shown promise towards more accurate, dense 3D scene maps. However, existing 3DGS-based methods fail to address the global consistency of the scene via loop closure and/or global bundle adjustment. To this end, we propose LoopSplat, which takes RGB-D images as input and performs dense mapping with 3DGS submaps and frame-to-model tracking. LoopSplat triggers loop closure online and computes relative loop edge constraints between submaps directly via 3DGS registration, leading to improvements in efficiency and accuracy over traditional global-to-local point cloud registration. It uses a robust pose graph optimization formulation and rigidly aligns the submaps to achieve global consistency. Evaluation on the synthetic Replica and real-world TUM-RGBD, ScanNet, and ScanNet++ datasets demonstrates competitive or superior tracking, mapping, and rendering compared to existing methods for dense RGB-D SLAM.
+</details>
+
+[📄 Paper](https://arxiv.org/abs/2408.10154) | [🌐 Project Page](https://loopsplat.github.io/) | [💻 Code](https://github.com/GradientSpaces/LoopSplat)
```
Comment (MrNeRF) [inline README.md:3046]: Can you please add an empty line after your changes and remove on before your entry?

**Review by MrNeRF — CHANGES_REQUESTED** [review]

**Comment by MrNeRF** [comment]
Thx for your contribution. Just a small change request before I merge.

**Comment by Zhu-Liyuan** [comment]
I've made the requested formatting changes. Please review. Thanks!

**Comment by MrNeRF** [comment]
Thank you!

**Pull request merged**